web/src/js/components/common/FileChooser.jsx

```diff
@@ -21,7 +21,7 @@ export default function FileChooser({ icon, text, className, title, onOpenFile }
                 ref={ref => fileInput = ref}
                 className="hidden"
                 type="file"
-                onChange={e => { e.preventDefault(); if(e.target.files.length > 0) onOpenFile(e.target.files[0]); fileInput = "";}}
```

Comment:
`fileInput = ""` should be changed to `fileInput.value = ""`
this call resets the fileInput. without this extra call it is not possible to open the same file twice...

fileInput.value = ""